src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java

```diff
@@ -46,6 +46,9 @@
     public static Module getSimpleModule()
     {
         SimpleModule module = new SimpleModule();
+        System.out.println ( "### Registering simple module (new snapshot)");
+        logger.info ( "### Registering simple module with logger.info");
+        logger.debug ( "### Registering simple module with logger.debug");
```

Comment:
:grinning:

Reply:
After your presentation lets try clearing the grape cache and seeing what does appear. I think my preference is for the `logger.info` for this case.

Reply:
Reverted to debug ; will deal with Jenkins separately.